src/commands/__tests__/toggleSort.ts

```diff
@@ -827,6 +827,54 @@ describe('store', () => {
   - a
   - b`)
     })
+
+    it('preserve sort order after multiple sort', async () => {
```

Comment:
For some reason this test passes even without the fix. This means that it doesn't actually cover the test case.